runtime/include/chpl-mem-desc.h

```diff
@@ -39,110 +41,94 @@
-          "string move data"),                                          \
-        m(STRING_SELECT_DATA,                                           \
-          "string select data"),                                        \
-        m(CONFIG_ARG_COPY_DATA,                                         \
-          "config arg copy data"),                                      \
-        m(CONFIG_TABLE_DATA,                                            \
-          "config table data"),                                         \
-        m(LOCALE_NAME_BUFFER,                                           \
-          "locale name buffer"),                                        \
-        m(TASK_DESCRIPTOR,                                              \
-          "task descriptor"),                                           \
-        m(TASK_DESCRIPTOR_LINK,                                         \
-          "task descriptor link"),                                      \
-        m(TASK_STACK,                                                   \
-          "task stack"),                                                \
-        m(MUTEX,                                                        \
-          "mutex"),                                                     \
-        m(LOCK_REPORT_DATA,                                             \
-          "lock report data"),                                          \
-        m(TASK_POOL_DESCRIPTOR,                                         \
-          "task pool descriptor"),                                      \
-        m(TASK_LIST_DESCRIPTOR,                                         \
-          "task list descriptor"),                                      \
-        m(THREAD_PRIVATE_DATA,                                          \
-          "thread private data"),                                       \
-        m(THREAD_LIST_DESCRIPTOR,                                       \
-          "thread list descriptor"),                                    \
-        m(IO_BUFFER,                                                    \
-          "io buffer or bytes"),                                        \
-        m(OS_LAYER_TMP_DATA,                                            \
-          "OS layer temporary data"),                                   \
-        m(GMP,                                                          \
-          "gmp data"),                                                  \
-        m(GETS_PUTS_STRIDES,                                            \
-          "put_strd/get_strd array of strides"),                        \
-        m(GETS_PUTS_COUNTS,                                             \
-          "put_strd/get_strd array of count"),                          \
-        m(NUM, "")                      // this must be the last entry
+#define CHPL_MD_ALL_MEMDESCS(m)                                               \
```

Comment:
Done! (Had gotten too caught up in the 8-space indentation and decided it wouldn't fit.)